scripts/tests/test_node_summary.py

```diff
@@ -56,21 +44,24 @@ def setUp(self):
         self.deleted_node = ProjectFactory(is_deleted=True)
         self.deleted_node2 = ProjectFactory(is_deleted=True)
 
-        self.date = datetime.datetime.utcnow() - datetime.timedelta(1)
+        self.date = timezone.now() - datetime.timedelta(1)
 
-        modify_node_dates_in_mongo(self.date - datetime.timedelta(0.1))
+        for node in AbstractNode.objects.all():
+            node.date_created = self.date - datetime.timedelta(0.1)
+            node.save()
+        # modify_node_dates_in_mongo(self.date - datetime.timedelta(0.1))
 
         self.results = NodeSummary().get_events(self.date.date())[0]
 
     def tearDown(self):
         super(TestNodeCount, self).tearDown()
-        Node.remove()
-        User.remove()
+        AbstractNode.remove()
```

Comment:
Should these be `.remove()` or `.objects.all().delete()`? (I don't know what MODM's `remove()` does)